src/tests/lvm/lvm_ha_controller/lvm_ha_controller_test.go

```diff
@@ -39,41 +38,25 @@ var nodeConfig lvm.LvmNodesDevicePvVgConfig
 var defLeaseSwitchTime = 120 // in seconds
 var nodesWithoutTaint []string
 var lvmControllerOrgReplica int32
+var app k8stest.FioApplication
 var volumeProvisionErrorMsg = "waiting for a volume to be created"
 
 func controllerHaTest(decor string, engine common.OpenEbsEngine, volType common.VolumeType, fstype common.FileSystemType, volBindModeWait bool) {
-
+	var err error
 	e2e_config := e2e_config.GetConfig().Product
-	app := k8stest.FioApplication{
+	app = k8stest.FioApplication{
 		Decor:                          decor,
-		VolSizeMb:                      4096,
+		VolSizeMb:                      1024,
 		OpenEbsEngine:                  engine,
 		VolType:                        volType,
 		FsType:                         fstype,
-		Loops:                          5,
+		Loops:                          7,
 		VolWaitForFirstConsumer:        volBindModeWait,
 		SkipPvcVerificationAfterCreate: true,
 	}
 
-	loopDevice := e2e_agent.LoopDevice{
-		Size:   10737418240,
-		ImgDir: "/tmp",
-	}
-
-	workerNodes, err := lvm.ListLvmNode(common.NSOpenEBS())
-	Expect(err).ToNot(HaveOccurred(), "failed to list worker node")
-
-	nodeConfig = lvm.LvmNodesDevicePvVgConfig{
-		VgName:        "lvmvg",
-		NodeDeviceMap: make(map[string]e2e_agent.LoopDevice), // Properly initialize the map
-	}
-	for _, node := range workerNodes {
-		nodeConfig.NodeDeviceMap[node] = loopDevice
-	}
-
-	logf.Log.Info("setup node with loop device, pv and vg", "node config", nodeConfig)
-	err = nodeConfig.ConfigureLvmNodesWithDeviceAndVg()
-	Expect(err).ToNot(HaveOccurred(), "failed to setup node")
+	// workerNodes, err := lvm.ListLvmNode(common.NSOpenEBS())
```

Comment:
Remove the commented code

Reply:
[fixed](https://github.com/openebs/openebs-e2e/compare/d92109e7bd8bcdec1e0dcaabcf44ae58ba75f223..0aeb8c6351a4007932ae44560cd4ce09885cbfb5)